kubetest2-gke/deployer/options/build.go

```diff
@@ -42,6 +42,7 @@ func (bo *BuildOptions) Validate() error {
 					BuildScript:   bo.BuildScript,
 					VersionSuffix: bo.CommonBuildOptions.VersionSuffix,
 					StageLocation: bo.CommonBuildOptions.StageLocation,
+					UpdateLatest:  bo.CommonBuildOptions.UpdateLatest,
```

Comment:
let's also plumb this value through to the KREL stager since it a commonoption ?

Reply:
Done